### PR TITLE
south bronx version of the glockroach

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

he got street smarts :)

mag dumping glockroach

## Why It's Good For The Game

South Bronx is an alright place

## Changelog
:cl:
add: glockroach has a very nice brother
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/glockroach.dm [hunk 1]
```diff
@@ -67,3 +67,42 @@
 /mob/living/simple_animal/hostile/glockroach/ex_act() //Explosions are a terrible way to handle a cockroach.
 	return
 
+#define AS_MANY_BULLETS_A_MAG_GOT 15
+
+#define REAL_NIGGA_HOURS 24 HOURS
```
Comment (81Denton) [inline code/modules/mob/living/simple_animal/hostile/glockroach.dm]: brug

**Comment by trollbreeder** [comment]
Gold slimes were a mistake.

File: code/modules/mob/living/simple_animal/hostile/glockroach.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/mob/living/simple_animal/hostile/glockroach.dm]: bruh

Comment (tralezab) [reply code/modules/mob/living/simple_animal/hostile/glockroach.dm]: i can undefine it but it'll be like the zimmerman case part 2

**Comment by Shadowflame909** [comment]
Glockroaches and South Bronx GR's should be enemy fractions

File: code/modules/mob/living/simple_animal/hostile/glockroach.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/modules/mob/living/simple_animal/hostile/glockroach.dm]: change it before github police finds it

File: code/modules/mob/living/simple_animal/hostile/glockroach.dm [hunk 2]
```diff
@@ -67,3 +67,42 @@
...
+
+/mob/living/simple_animal/hostile/glockroach/magdump/Shoot(atom/targeted_atom, bullet_count)
+	. = ..()
+	if(bullet_count = AS_MANY_BULLETS_A_MAG_GOT)
+		reloading_n_shit()
+
+/mob/living/simple_animal/hostile/glockroach/magdump/proc/reloading_n_shit()
+	to_chat(src, "<span class='warning'>YOU EMPTY SON, RELOADING</span>")
+	var/obj/item/ammo_box/magazine/m45/ohnine/mag = new(get_turf(src))
+	addtimer(CALLBACKCALLBACK(src, .proc/donereloadin) 10 SECONDS)
```
Comment (spookydonut) [inline code/modules/mob/living/simple_animal/hostile/glockroach.dm]:
```suggestion
	addtimer(CALLBACK(src, .proc/donereloadin), 10 SECONDS)
```

**Comment by trollbreeder** [comment]
all you had to do was follow the damn clown car, CJ

**Comment by mrhugo13** [comment]
unironically the best PR this year.

**Comment by trollbreeder** [comment]
uh oh
the fun police are here

**Comment by SpinnerMaster** [comment]
wack

**Comment by AtlasRules** [comment]
uziroach and glock 18 roach when

**Comment by PKPenguin321** [comment]
you best hope the github cops dont see this

**Pull request closed without merging**